### ISSUE TITLE
Issues on running 'npm start' | Apple M1

### ISSUE THREAD

**Issue by riolet-vose** [HabitRPG/habitica#13770]
### General Info
  * OS: Mac OS Monterey v12.1 on Apple Silicon
  * Node.js : v14.18.2
  * npm : 6.14.15

### Description
Followed the instructions published here - 
https://habitica.fandom.com/wiki/Setting_up_Habitica_Locally_on_MacOS 

Getting the following error when I run npm start :-

```
> habitica@4.218.0 start /Users/--/--/--/Habitica/habitica
> gulp nodemon

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/--/--/--/Habitica/habitica/node_modules/gulp-imagemin/index.js
require() of ES modules is not supported.
require() of /Users/--/--/--/Habitica/habitica/node_modules/gulp-imagemin/index.js from /Users/--/--/--/Habitica/habitica/gulp/gulp-sprites.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/--/--/--/Habitica/habitica/node_modules/gulp-imagemin/package.json.

    at new NodeError (internal/errors.js:322:7)
    at Module._extensions..js (internal/modules/cjs/loader.js:1102:13)
    at Object.newLoader [as .js] (/Users/--/--/--/Habitica/habitica/node_modules/pirates/lib/index.js:141:7)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/Users/--/--/--/Habitica/habitica/gulp/gulp-sprites.js:2:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Module._compile (/Users/--/--/--/Habitica/habitica/node_modules/pirates/lib/index.js:136:24) {
  code: 'ERR_REQUIRE_ESM'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! habitica@4.218.0 start: `gulp nodemon`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the habitica@4.218.0 start script.
```

#### Console Errors

On running npm audit -
```
npm WARN acorn-import-assertions@1.8.0 requires a peer of acorn@^8 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb-base@14.2.0 requires a peer of eslint@^5.16.0 || ^6.8.0 || ^7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-habitrpg@6.2.0 requires a peer of eslint@^6.5.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-import@2.22.0 requires a peer of eslint@^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-vue@5.2.3 requires a peer of eslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN vue-eslint-parser@5.0.0 requires a peer of eslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by riolet-vose** [comment]
@CuriousMagpie @SabreCat What can be done?
I have reported this using e-mail id 2000raspi@gmail.com

**Comment by dnow2000** [comment]
hello, I saw in the console errors that it is telling you that you are missing peer dependencies. I found this link to stack overflow that shows someone that has a similar problem:
https://stackoverflow.com/questions/46053414/npm-warn-requires-a-peer-of-but-none-is-installed-you-must-install-peer

Use following for each peer dependency to install that and remove the error:
npm install --save-dev xxxxx

**Comment by CuriousMagpie** [comment]
@sahyrlelyaams Are you still having this problem? As @dnow2000 points out, the issue is with your peer dependencies. Did you run `sudo npm install -g mocha` after installing node and npm? I'm using an Apple M1 and haven't had any issues running a local copy of the application.

**Comment by riolet-vose** [comment]
> @sahyrlelyaams Are you still having this problem? As @dnow2000 points out, the issue is with your peer dependencies. Did you run `sudo npm install -g mocha` after installing node and npm? I'm using an Apple M1 and haven't had any issues running a local copy of the application.

@CuriousMagpie I have globally installed mocha before running npm install.

As for installing the peer dependencies, I went through a never ending rabbit hole as after installing eslint & acorn packages, which were supposedly missing, it said popper.js is missing, which I tried to install as well. 

I installed popper.js using --save-dev & now, it still can't detect the installation of popper.js for some reason. I ran npm ci as well, with not much success.
